kubefwd/sudo-kubefwd.sh

```
@@ -27,6 +29,32 @@ if [[ "$ENTR" == "" ]]; then
     exit 1
 fi
 
+# Initialize the namespaces we need to watch.
+# Write them to the file that we use to trigger reloads.
+NAMESPACES=$(tilt get kd -o=jsonpath='{.items[*].spec.watches[*].namespace}' | tr -s ' ' '\n' | sort -u)
+echo "$NAMESPACES" > "$TRIGGER"
+chmod a+rw "$TRIGGER"
+
+"$DIR/watch-namespaces.sh" "$TRIGGER" &
+WATCH_PID="$!"
+
+function cleanup {
+    set -x
+    set +e
+
+    # Remove the trigger file. This will make any dangling entr/kubefwd
+    # processes exit. (We can't kill them directly b/c they're running with sudo
+    # privs.)
+    rm "$TRIGGER"
+    
+    kill "$WATCH_PID"
+    wait "$WATCH_PID"
+
+    # Remove the trigger file twice, in case the watcher created a new one.
+    rm "$TRIGGER"
```

If this only happens sometimes, it should probably be `rm -f` to ignore errors